CONTRIBUTING.md

```diff
@@ -101,3 +98,20 @@ In case you **don't have an idea of a quest?** Look [at the existing issues](htt
 Always remember to pay attention to [the new quest guide](https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete) of StreetComplete! It also has tips for implementing a quest.
 
 You can look at **other quests** to see how they are implemented.
+
+### StreetComplete-related projects
+
+#### Dependencies
+
+* [Tangram-ES](https://github.com/tangrams/tangram-es/) for technical issues with the map rendering
+* [countryboundaries](https://github.com/westnordost/countryboundaries) for detecting, in which country a quest is (affects quest display, etc.)
+* [osmapi](https://github.com/westnordost/osmapi) for communication with the OSM API
+
+#### Created for StreetComplete
+
+* [StreetCompleteJSON](https://github.com/goldfndr/StreetCompleteJSON) by [@goldfndr](https://github.com/goldfndr) listing all tags edited by StreetComplete for [taginfo](https://taginfo.openstreetmap.org/projects/streetcomplete)
+* [blacklistr](https://github.com/ENT8R/blacklistr) by [@ENT8R](https://github.com/ENT8R) for visualizing StreetComplete's country exclusion list
+* [NotesReview](https://github.com/ENT8R/NotesReview) by [@ENT8R](https://github.com/ENT8R) for reviewing notes with a specific keyword (here: `StreetComplete`)
+* [streetcomplete-mapstyle](https://github.com/ENT8R/streetcomplete-mapstyle) by [@ENT8R](https://github.com/ENT8R) maintaining the mapstyle of StreetComplete
+* [StreetCompleteQuestCreator](https://github.com/ENT8R/StreetCompleteQuestCreator) by [@ENT8R](https://github.com/ENT8R) for creating simple quests with a web interface
```

Comment:
I think this can be removed... I created some quests with that tool and it is no help at all because you will have to change something afterwards in nearly every case. Especially for new contributors who want to create a new quest easily without programming knowledge this is a huge problem... Some people sent me their quests so I can look if they did everything right and many were not aware of the special (but simpler) query syntax. So in the end this tool creates more (unnecessary) work if somebody else has to look at the code and correct more than necessary.
And another reason to better not include this project are the other problems this tool has: #959 is a quest I generated with this tool to test it. But all images really looked like "scaled with MSPaint" 😄 so I had to replace all images by hand afterwards.
And I do not really want to improve this logic of (re)scaling because I fear this might be too much effort...

So I think this project is not worth it to be included here.

Reply:
Ah, okay, so you really like to abandon that project. In this case, please also state this in your GitHub repo there (Readme) and maybe [archive that repo](https://help.github.com/articles/archiving-a-github-repository/).

Reply:
Alright. Thank you for the hint!